libkbfs/file_data.go

```diff
@@ -65,13 +65,21 @@ type parentBlockAndChildIndex struct {
 	childIndex int
 }
 
+func (pbci parentBlockAndChildIndex) childIPtr() IndirectFilePtr {
+	return pbci.pblock.IPtrs[pbci.childIndex]
+}
+
+func (fd *fileData) rootBlockPointer() BlockPointer {
```

Comment:
mind splitting this (and the above) off into a separate PR? It would make subsequent passes easier.